.github/workflows/code-coverage.yml

```
@@ -39,25 +37,72 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-composer-
 
+      # As suggested here https://github.com/codeclimate/test-reporter/issues/406#issuecomment-578483422
+      - name: Set ENV for CodeClimate (pull_request)
+        if: github.event_name == 'pull_request'
+        run: |
+          echo $GITHUB_HEAD_REF >> $GIT_BRANCH
+          echo "$(git rev-parse origin/$GITHUB_HEAD_REF)" >> $GIT_COMMIT_SHA
+
+      - name: Set ENV for CodeClimate (push)
+        if: github.event_name == 'push'
+        run: |
+          echo $GITHUB_REF >> $GIT_BRANCH
+          echo $GITHUB_SHA >> $GIT_COMMIT_SHA
```

This does not seem right, and doesn't seem to match what the linked thing suggests.

Did you mean something like this?
```suggestion
      # As suggested here https://github.com/codeclimate/test-reporter/issues/406#issuecomment-578483422
      - name: Set ENV for CodeClimate (pull_request)
        if: github.event_name == 'pull_request'
        run: |
          echo "GIT_BRANCH=$GITHUB_HEAD_REF" >> $GITHUB_ENV
          echo "GIT_COMMIT_SHA=$(git rev-parse origin/$GITHUB_HEAD_REF)" >> $GITHUB_ENV

      - name: Set ENV for CodeClimate (push)
        if: github.event_name == 'push'
        run: |
          echo "GIT_BRANCH=$GITHUB_REF" >> $GITHUB_ENV
          echo "GIT_COMMIT_SHA=$GITHUB_SHA >> $GITHUB_ENV
```

---

I see b50e0d4 already fixed it.

---

yeah, I noticed it earlier. Should be fine as of b50e0d4